enrollment-server-onboarding/pom.xml

```diff
@@ -157,6 +157,13 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
I would rather use only `maven.deploy.skip` https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html#skip but prefer to be consistent.

Reply:
We can change it one day in all projects.